README.md

```diff
@@ -1,13 +1,208 @@
+- **`substitutionPattern`**: Substitutions allow you to include variables of
+  your execution environment in the config. For example, assignments like
+  `key: ${MY_VALUE}` in your config file can be substituted at runtime, resulting
+  in `key: process.env['MY_VALUE']`. The `substitutionPattern` is an escaped
+  regexp string that determines the format of substituted values in your config.
+  In the given example and by default, this is of form `${<my-var>}`.
+- **`configPath`**: The path to the directory that contains one or multiple
+  config files.
+- **`schemaPath`**: The path the schema file that contains valid
+  [`json-schema`](https://json-schema.org/).
+
+## Schema Validation
+
+Besides writing config files, you can define a schema file which can be used
+to validate your configs. The schema file must be written in JSON according to the
+[`json-schema`](https://json-schema.org/) standard. To get started, you can have
+a look at the [official learning resources](https://json-schema.org/learn/) or
+checkout the examples in the `/example` directory of this project.
+
+You can provide the path to your schema file via `options.schemaPath`. Please
+note that this file must be named `schema.json` if you decide to place it in
+the same directory as your config files (which is the default).
+
+However, `strong-config` will work fine if you decide to not use schemas at all.
+
+## FAQ
+
+1. **Can I write my configs as JSON files?**
+
+   Yes, YAML and JSON are supported. Please pay attention to the file extensions
+   (`.yaml|yml` or `.json` are allowed).
+
+1. **Do you support schemas other than [`json-schema`](https://json-schema.org/)?**
+
+   No, currently we only support a schema based on `json-schema`.
+
+1. **Can I use `@strong-config/node` for the browser, too?**
+
+   No, single page applications (SPAs) are not suitable for containing secrets.
+   This is because SPA code is delivered to the client and is therefore always public.
+   TODO: explain what you can do when your configs don't have secrets.
+
+1. **Can I use `@strong-config/node` for Server-Side-Rendered (SSR) apps, too?**
+
+   It depends. The server-side can make use of `@strong-config/node` to manage
+   secrets. However, the client-side can't because of the reason in the previous
+   point.
+
+1. **Can my config contain arrays?**
+
+   Yes, but not as top-level definition. The top-level of any config file must be
+   an object so SOPS can store its metadata in the field `sops`. Otherwise, you
+   can define arrays at any nested level of the config file.
+
+## Development and Contributions
+
+We added a few scripts (under `/scripts/dev` with entrypoints in `package.json`)
+for development purposes:
+
+- `yarn dev:load`/`yarn dev:load:es6`, and `yarn dev:load:commonjs`
+- `yarn dev:validate`
+
+These use an exemplary config setup in `/example`.
 
-...
+Our contribution guidelines are posted in `CONTRIBUTING.md`
```

Comment:
This won't be true until we actually have a `CONTRIBUTING.md`, I'd delete it until we actually have it, otherwise it's just a dead link

Reply:
right. deleted